code/api/src/modules/user/tests/userQuery.test.js

```diff
@@ -17,13 +17,16 @@ describe('User queries', () => {
 
         res.body.data.user.should.have.property('email')
         expect(res.body.data.user.email).to.eq('user@crate.com')
+
+        res.body.data.user.should.have.property('description')
+        expect(res.body.data.user.description).to.eq('Tattooed seitan waistcoat austin asymmetrical chambray hot chicken man bun poke')
```

Comment:
I love that our new tests are now additions to our user query or user mutation. This make readability so much easier and it is pretty easy to understand